### ISSUE TITLE
React Warning with Searchbox

### ISSUE THREAD

**Issue by richtera** [searchkit/searchkit#691]
I think it should either use defaultValue or an empty onChange handler or something like that. Not quite sure why it's complaining now as opposed to when we first started using it.
```
Warning: Failed prop type: You provided a `value` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultValue`. Otherwise, set either `onChange` or `readOnly`.
    in input (created by SearchBox)
    in form (created by SearchBox)
    in div (created by SearchBox)
    in SearchBox (created by HomePage)
    in div (created by HomePage)
    in div
```

**Comment by devtanuj** [comment]
![Screen Shot 2019-05-13 at 3 38 31 PM](https://user-images.githubusercontent.com/47532343/57613728-3cac1c00-7595-11e9-852b-77070ada8a75.png)
 

it seems value is being set to empty and not being controlled by the react

**Comment by devtanuj** [comment]
this issue is being resolved in  "searchkit": "^2.3.1-alpha.9",

**Comment by joemcelroy** [comment]
closing issue as fixed in latest alpha release.
